docs/attempts-api/schemas/events/account-recovery/ForgotPasswordEmailRateLimited.yml

```diff
@@ -0,0 +1,9 @@
+description: |
+  User has exceeded the number of email reset notifications (20 attempts within 60 mins) that can be sent to one user’s email addresses.
```

Comment:
Back in the day&trade;, I argued against (but unsuccessfully) documenting rate limits in the schema. I'd like to re-introduce that suggestion, because:
1. I don't think they are meant to be public information (not that it would be too difficult to probe out, admittedly), and
2. They're subject to change at any time, and likely by someone other than those of us maintaining these docs, so it's not easy to even be sure our documentation is correct.

Reply:
I agree with ☝️, mostly because the actual rate limit numbers will likely go stale here

Reply:
oh sure, i don't feel strongly that this should be documented, and did not check a single one, so they're probably inaccurate anyways